docs/Manage-Space.rst

```diff
@@ -227,31 +227,37 @@ follows:
 
 1. Click |image22| at the top left corner of the **Space** page to open the Add New Space form.
 
-2. In the Settings tab, enter a space name (required and from 3 to 200 characters) 
-   and description (from 0 to 255 characters).
+2. A popup is displayed containing all informations related to the space creation.
 
 |image23|
 
+3. The space creator must enter a space name (3 to 200 characters) 
+   and may enter an optional description (max 255 characters).
```

Comment:
I think that we should precise the minimum characters which is 0 for description i.e. it not mandatory to fill it.

Reply:
While we have "optional" description, it means that the minimum is 0.